chore: update links (#1047)
dfinity/portal: docs/developer-docs/frontend/index.md

---
sidebar_position: 1
---

# Building a web frontend

The Internet Computer allows you to host frontends built with standard web technologies for your dApps, using
our [JavaScript agent](https://www.npmjs.com/package/@dfinity/agent) as a communication layer. By using

> the [asset canister](https://github.com/dfinity/certified-assets) provided by `dfx` to upload static files to the IC,

Reviewer: ```suggestion
the [asset canister](https://github.com/dfinity/sdk/tree/master/src/canisters/frontend/ic-frontend-canister) provided by `dfx` to upload static files to the IC,
```
The asset canister has moved

Starting with dfx 0.7.7, we now provide you with webpack dev-server in our `dfx new` starter.

The webpack development server—`webpack-dev-server`—provides in-memory access to the webpack assets, enabling you to
make changes and see them reflected in the browser right away using live reloading.

To take advantage of the `webpack-dev-server`:

1. Create a new project and change to your project directory.

2. Start the IC locally, if necessary, and deploy as you normally would, for example, by running the `dfx deploy`
   command.

3. Start the webpack development server by running the following command:

       npm start

> 4. Open a web browser and navigate to the asset canister for your application using port 8080.

Reviewer: ```suggestion
4. Open a web browser and navigate to the asset canister for your application using port 4943.
```
new default port

   For example:

>        http://localhost:8080

Reviewer: ```suggestion
       http://localhost:4943
```
